test/RibbonThetaVault.js

```diff
@@ -2426,6 +2426,98 @@ function behavesLikeRibbonOptionsVault(params) {
       });
     });
 
+    describe("#withdrawLater", () => {
```

Comment:
After a `withdrawLater` call is successful, we need to check that the `scheduledWithdrawals[msg.sender]` value is set correctly.

We also need to check that the `ScheduleWithdraw(msg.sender, shares)` event has been emitted correctly